analysers/analyser_merge_power_line_FR.py

```diff
@@ -47,8 +47,13 @@ def __init__(self, config, logger = None):
                     tags = [{'power': 'line'}, {'disused:power': 'line'}]),
                 conflationDistance = 30,
                 mapping = Mapping(
-                    static1 = {'power': 'line'},
-                    static2 = {'source': self.source},
+                    static1 = {
+                        'power': 'line',
+                        'operator': 'RTE'
```

Comment:
Maybe here adding operator in static1 is right ?

Reply:
Yes that's right here.
Only 2 261 power=line are missing operator in France and they are not covered by RTE dataset. It may generate ~1k new issues, probably less.